api/nodes/serializers.py

```diff
@@ -600,6 +601,9 @@ def get_region_id(self, obj):
             region_id = obj.osfstorage_region._id
         return region_id
 
+    def get_wiki_enabled(self, obj):
+        return obj.has_addon('wiki')
```

Comment:
This adds a database query per node, on every list of nodes. Seems like we already struggle with the N+1 problem and I worry about adding work to super common API endpoints. That said, I'm not sure what's a good way around it here... maybe always include the wiki relationship but put something in its response to indicate it's disabled?

Reply:
Talked about this in person. Unless someone on platform objects, I think we can just leave this as an attribute on the node and possibly refactor later to have it be a cached attribute rather than calculated. Or, you know, Varnish.

Reply:
>  I think we can just leave this as an attribute on the node 

Can you clarify this?

Reply:
> I think we can just leave this as a computed attribute on the node serializer

Reply:
Also, it should be noted that this calculation is already occurring on every list of nodes:
https://github.com/CenterForOpenScience/osf.io/blob/1d11c3b11ea3b2dc2fa2797bdf779ae05b8c87ea/api/nodes/serializers.py#L341
https://github.com/CenterForOpenScience/osf.io/blob/1d11c3b11ea3b2dc2fa2797bdf779ae05b8c87ea/api/base/serializers.py#L200-L208
but now that I look at this, I see it first tries `has_wiki_addon`, which is an optimization:
https://github.com/CenterForOpenScience/osf.io/blob/1d11c3b11ea3b2dc2fa2797bdf779ae05b8c87ea/api/nodes/utils.py#L72-L84
so it should calculate it using that, if available.